src/run_releasenotes_check.py

```diff
@@ -23,18 +25,33 @@
 def main() -> int:
     args = ReleaseNotesCheckArgs()
     console.configure(level=args.logging_level)
-    manifest_file = InputManifest.from_file(args.manifest)
-    BUILD_VERSION = manifest_file.build.version
-
+    manifests: List[InputManifest] = []
     # storing temporary release notes for testing purposes
     BASE_FILE_PATH = "release_notes_workflow/results"
+
+    for input_manifests in args.manifest:
+        manifests.append(InputManifest.from_file(input_manifests))
+
+    #  Assuming that the OS and OSD manifests will be provided for the same release version
+    BUILD_VERSION = manifests[0].build.version
```

Comment:
Can we do a check between OS and OSD if user input both, and error out if it is not matching?

Reply:
Added checks.